plugins/source/aws/resources/services/cloudformation/template_summaries.go

```diff
@@ -0,0 +1,56 @@
+package cloudformation
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+)
+
+func templateSummaries() *schema.Table {
+	tableName := "aws_cloudformation_template_summaries"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_GetTemplateSummary.html`,
+		Resolver:    fetchTemplateSummary,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "cloudformation"),
+		Transform: transformers.TransformWithStruct(
+			&cloudformation.GetTemplateSummaryOutput{},
+			transformers.WithSkipFields("ResultMetadata"), // This field contains metadata about the API call rather than the template itself, so remove it.
+		),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:     "stack_id",
+				Type:     schema.TypeString,
+				Resolver: schema.ParentColumnResolver("id"),
+			},
+			{
+				Name:     "metadata",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Metadata"),
+			},
```

Comment:
I think bringing the CF stack ARN in as a column here would be useful, and if I'm not mistaken it could also serve as a primary key. We can do something like this:

```
{
   Name:     "stack_arn",
   Type:     schema.TypeString,
   Resolver: schema.ParentColumnResolver("arn"),
   CreationOptions: schema.ColumnCreationOptions{
	PrimaryKey: true,
  },
},
```

What do you think?